### PR TITLE
Add support for get_route_to and get_lldp_neighbors to PANOS

### PR DESCRIPTION
**Pull request by XioNoX**
This PR adds support for get_route_to and get_lldp_neighbors to PAN firewalls.

I might need some help on fixing the tests.

It seems like LLDP and routes behave the same way. If the device returns only one <entry> for a given query, this entry will be converted to a dict. While if there is several <entry> it will be a list of dicts.
The 2nd commit ("Better handling of single items lists") handles that properly. It has been tested with get_route_to but not get_lldp_neighbors.

Be aware that there is at least one minor issue with pan-python, if the lldp neighbor has special characters (such as "<") in its description. Opened https://github.com/kevinsteves/pan-python/issues/14

Please let me know what you think!

### CODE REVIEW

**Comment by dbarrosop** [comment]
Thanks for your contribution. Seems that tests are not passing due to missing mocked data. Could you take a look, please?

File: napalm_panos/panos.py [hunk 1]
```diff
@@ -378,6 +378,119 @@ def get_facts(self):
...
+
+            if local_int not in neighbors.keys():
+                neighbors[local_int] = []
+
+            lldp_neighs = lldp_item['neighbors']['entry']
+            if isinstance(lldp_neighs, dict):
+                single_item_list = []
+                single_item_list.append(lldp_neighs)
+                lldp_neighs = single_item_list
+
```
Comment (GGabriele) [inline napalm_panos/panos.py]: You can simplify this with:

```python
if isinstance(lldp_neighs, dict):
    lldp_neighs = [lldp_neighs]
```

Comment (XioNoX) [inline napalm_panos/panos.py]: Thanks, updated.

**Comment by dbarrosop** [comment]
@GGabriele can you review and release when you can?

**Comment by dbarrosop** [comment]
Oh, I just saw you were on it. Sorry.

**Comment by GGabriele** [comment]
Everything looks good to me. Thanks @XioNoX :)

**Pull request merged**